src/js/api.js

```diff
@@ -56,6 +56,7 @@ export function sendFetch(options = {method: 'GET'}){
         return response.text()
       }
       const json = response.json()
+      console.log('got api response for', url, json)
```

Comment:
Can we keep this in? I think it pairs well with the log above it without being too spammy